cmd/app/command/start.go

```diff
@@ -46,6 +47,7 @@ func NewCmdStart() *cobra.Command {
 			metrics.StartingTime = time.Now()
 			// Start HTTP server for /metrics endpoint
 			go server.StartHttpServer()
+			go udevevent.MonitorEvent()
```

Comment:
we may miss a disk  attach event when disk is atttached after pushDiskResources in the main thread and before MonitorEvent calls *select* for the netlink fd in go routine. Probably we should use epoll where after epoll_create and epoll_ctl, we can notify main thread that it is now listening for the netlink fd and it can proceed with the pushDiskResources , in that way we will not miss any attach/detach event.

Reply:
I have created one issue for this (#50 )